publisher-sdk/src/main/java/com/criteo/publisher/model/CacheAdUnit.kt

```diff
@@ -13,18 +13,14 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+package com.criteo.publisher.model
 
-package com.criteo.publisher.util;
+import com.criteo.publisher.annotation.OpenForTesting
+import com.criteo.publisher.util.AdUnitType
 
-import androidx.annotation.NonNull;
-import com.criteo.publisher.model.CdbRequest;
-import com.criteo.publisher.model.CdbResponse;
-
-public interface CdbCallListener {
-
-  void onCdbRequest(@NonNull CdbRequest request);
-
-  void onCdbResponse(@NonNull CdbRequest request, @NonNull CdbResponse response);
-
-  void onCdbError(@NonNull CdbRequest request, @NonNull Exception exception);
-}
+@OpenForTesting
+data class CacheAdUnit(
+    val size: AdSize?,
```

Comment:
Basically making non-nullable will make a few tests in `AdUnitMapperTests` fail, because AdUnitMapper will try to construct `CacheAdUnit` object with null items.